builder/processor/src/main/java/io/helidon/builder/processor/PrototypeProperty.java

```diff
@@ -115,6 +116,10 @@ static PrototypeProperty create(ProcessingContext processingContext,
         );
     }
 
+    public Field.Builder fieldDeclaration(boolean isBuilder) {
```

Comment:
not needed `public`, this is package private record.

Reply:
Fixed